overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/module-setup.sh

```diff
@@ -36,5 +36,5 @@ install() {
     # note we `|| exit 1` here so we error out if e.g. the units are missing
     # see https://github.com/coreos/fedora-coreos-config/issues/799
     systemctl -q --root="$initdir" add-requires "$target" rhcos-fips.service || exit 1
```

Comment:
Can you inline the variable from 2 lines above and remove the now stale comment?